### ISSUE TITLE
findNumbers with extended doesn't detect like parseNumber

### ISSUE THREAD

**Issue by hereismass** [catamphetamine/libphonenumber-js#209]
Hello,

I am trying to detect numbers in a paragraph that are no necessary valid (lot of people don't put the `+` in front of the country code).
Using `findPhoneNumbers` doesn't find anything whereas `parseNumber` does:

With an 'invalid' french number:
```
parseNumber('33174563521', 'US', {extended: true })); // => found
parseNumber('33174563521', 'FR', {extended: true })); // => found

findPhoneNumbers('33174563521', 'US', {extended: true })); // nothing
findPhoneNumbers('33174563521', 'FR', {extended: true })); // nothing
```

Is there something I did not understood in the usage of the lib or is this a bug?

Using v1.2.2

Thanks :)

**Comment by catamphetamine** [comment]
One could say that it's a bug: the options weren't passed to `parse()`.
Released 1.2.3 with the fix.

**Comment by hereismass** [comment]
Thanks! Works perfectly now :)
